nearai/agents/models/tool_definition.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+from pydantic import BaseModel, ConfigDict
+
+class MCPTool(BaseModel):
```

Comment:
this works for now, but for the future we should try to import this definition from the official mcp python sdk. Can we put a TODO comment here above the class so we know to fix this?